### PR TITLE
Partial out footer and header.

### PR DESCRIPTION
**Pull request by sowiecki**
Yay preventing merge conflicts. I will work on the header to complete the donation integration.

### CODE REVIEW

**Comment by thewildpendulum** [comment]
Settle on how many spaces per tab you're using.

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<footer>
+	<h5>Humble Hearts</h5>
```
Comment (thewildpendulum) [inline app/views/layouts/_footer.html.erb:2]: Use 2 spaces

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+</footer>
```
Comment (thewildpendulum) [inline app/views/layouts/_footer.html.erb]: Newline at end of file

**Comment by schmidtsusanr** [comment]
Looks good to me

**Comment by sowiecki** [comment]
@thewildpendulum @wiliajc87

**Comment by thewildpendulum** [comment]
@Nase00 :+1: 
@wiliajc87 Please merge

**Comment by thewildpendulum** [comment]
Wait. I lied. This needs to be merged into `develop`

**Pull request closed without merging**